h2o-core/src/main/java/water/H2O.java

```diff
@@ -2171,7 +2183,7 @@ public static void main( String[] args ) {
     ArrayList<String> args2 = new ArrayList<>(Arrays.asList(args));
     for( Object p : System.getProperties().keySet() ) {
       String s = (String)p;
-      if( s.startsWith("ai.h2o.") ) {
+      if( s.startsWith("ai.h2o.") && !IGNORED_PROPERTIES.contains(s)) {
```

Comment:
Suggestion: The h2o arguments never have a dot `.` in the name, can we ignore all properties that start with `ai.h2o.` but do have a dot character after this initial prefix?

That would solve this problem for all other cases as well. Not just Jetty.

Reply:
Added. It is assumend all arguments have `ai.h2o.` prefix.